tests/unit/test_core_greeting.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the core.greetings module"""
+
+from typing import Optional, Type, Any
+from contextlib import nullcontext
+import pytest
+from my_microservice.core.greeting import generate_greeting
+from my_microservice.models import Greeting
+from .fixtures.greetings import greeting_expression_fixture
+
+
+@pytest.mark.parametrize(
+    "language,isinformal,exception_expected",
+    [
+        ("Greek", True, None),
+        ("Greek", False, None),
+        ("Croatian", True, ValueError),
```

Comment:
So Croatian does not have informal greetings, so if you get one that is, you get a Value Error?

Reply:
Yes exactly, so of course Croatian has informal greetings, but we haven't configured them this is why we expect a value error. Again just a stupid example for this toy service.